api/docs/release.dox

```diff
@@ -157,6 +157,9 @@ Further non-compatibility-affecting changes include:
  - Allow v2p.textproto file in a trace directory. This file is present in public traces.
  - Allow v2p.textproto file to have one missing virtual_address field, which indicates
    virtual_address == 0x0. Necessary in case a trace accesses virtual address 0x0.
+ - Added noise_generator_t scaffolding as a reader_t to produce synthetic trace records.
```

Comment:
This is the 11.3 list: the next version list is up above.

Reply:
Fixed, thanks.